src/theme/sass-partials/_input.scss

```diff
@@ -1,4 +1,5 @@
-.vehicle-reg-input {
+.vehicle-reg-input,
+.pass-cert-input {
```

Comment:
@johnmorleybjss Should we make the input field a generic class?

Reply:
I was thinking we should probably just make all text inputs take this style without the need for an additional class with an attribute value selector, do you agree?